### PR TITLE
srm rn for 792xx

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/whats-new.md [hunk 1]
```diff
@@ -17,7 +17,73 @@ Review the notes below to learn about the new features that are Generally Availa
...
-## Latest - April 22, 2023, version 79111
+## Latest - April xx, 2023, version 792xx
+
+### Service Reliability Management
+
+- An icon appears on the SLO performance trend chart timeline to indicate when the error budget was reset and the amount of budget that was added. (SRM-14550)
+
+### Harness Delegate 
+
+- Added the following metrics for immutable delegates that you can scrape via Prometheus: (DEL-5363)
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:28]: I'm not sure about the usage of via. Can we say scrape by using Prometheus?

**Review by roshnisarangadharan — APPROVED** [review]
Approved with one minor suggestion. Ignore if irrelevant.

File: release-notes/service-reliability-management.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Service Reliability Management release notes
 sidebar_label: Service Reliability Management
-date: 2023-04-10T10:00
+date: 2023-05-02T10:00:20
```
Comment (SudheendraKatte) [inline release-notes/service-reliability-management.md:4]:
```suggestion
date: 2023-05-04T10:00:20
```

File: release-notes/service-reliability-management.md [hunk 3]
```diff
@@ -15,16 +15,14 @@ Review the notes below for details about recent changes to Harness Service Relia
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest - April 22, 2023, version 79111
+## Latest - May xx, 2023, version 792xx
```
Comment (SudheendraKatte) [inline release-notes/service-reliability-management.md:18]:
```suggestion
## Latest - May 04, 2023, version 79214
```

**Comment by SudheendraKatte** [comment]
Content has already been reviewed and approved. Updated the date and release number.

**Pull request merged**